### ISSUE TITLE
linting should prompt for action when 1:1s are declined by the other person (or all other people)

### ISSUE THREAD

**Issue by flavorjones** [flavorjones/calendar-assistant#8]
The web UI does a thing where it shows when the other person has declined.

I'd like linting to prompt me to decline or reschedule.

An imaginary journey:

```
$ calendar-assistant lint today
Found meetings that were declined:

2018-09-17 08:00 - 09:00  | 👫 MD / RDZ (1:1, recurring)
Would you like to reschedule? (y/n): n
(Declining  "👫 MD / RDZ")

2018-09-17 14:00 - 14:30  | 👫 Evan / MikeD (1:1, recurring)
Would you like to reschedule? (y/n): y
(Rescheduling "👫 Evan / MikeD")
... eliding this input because it's another story ...
```

(Depends on #7 for rescheduling functionality.)
